server/channels/app/notification_push.go

```diff
@@ -635,10 +704,13 @@ func (a *App) SendTestPushNotification(deviceID string) string {
 	case model.PushStatusRemove:
 		return "false"
 	case model.PushStatusFail:
-		a.NotificationsLog().Error("Notification error",
-			mlog.String("type", msg.Type),
-			mlog.String("deviceId", msg.DeviceId),
-			mlog.String("status", pushResponse[model.PushStatusErrorMsg]),
+		a.NotificationsLog().Error("Push proxy failed to send test notification",
+			mlog.String("type", model.TypePush),
+			mlog.String("push_type", msg.Type),
+			mlog.String("status", model.StatusServerError),
+			mlog.String("reason", model.ReasonPushProxyError),
+			mlog.String("device_id", msg.DeviceId),
+			mlog.String("push_response", pushResponse[model.PushStatusErrorMsg]),
```

Comment:
Would it be better to rename this key to be "error" and use `mlog.Err`?

Reply:
Does that work? I think `pushResponse[model.PushStatusErrorMsg]` is a `string`, unless you're wanting me to wrap a new error around this.